### PR TITLE
Prevent tag submission if constraints are not fullfiled

### PR DESCRIPTION
**Pull request by noirbizarre**
This PR ensure that the tag completer component does not allow to create a tag not fullfilling the min and max length constraints.

### CODE REVIEW

File: js/components/form/tag-completer.vue [hunk 1]
```diff
@@ -11,11 +12,16 @@ export default {
...
         valueField: 'text',
         plugins: ['remove_button'],
-        create: function(input) {
+        create(input) {
             return {
                 value: input,
                 text: input
             }
+        },
+        createFilter(value) {
```
Comment (vinyll) [inline js/components/form/tag-completer.vue]:
```
createFilter(value) {
  return value && config.tags.MIN_LENGTH <= value.length <= config.tags.MAX_LENGTH;
```

File: udata/tags.py [hunk 2]
```diff
@@ -3,6 +3,7 @@
 
 from slugify import slugify
 
+# Constraints. Should be synced with js/config.js
```
Comment (davidbgk) [inline udata/tags.py]: Why not now?

File: js/components/form/tag-completer.vue (hunk 1 above)
Comment (noirbizarre) [inline js/components/form/tag-completer.vue]: already tried but not working 😢

File: udata/tags.py (hunk 2 above)
Comment (noirbizarre) [inline udata/tags.py]: They are, it's just a reminder for futur modifications

File: js/components/form/tag-completer.vue (hunk 1 above)
Comment (vinyll) [inline js/components/form/tag-completer.vue]: ?
What about 
```
if (value) {
  return config.tags.MIN_LENGTH <= value.length <= config.tags.MAX_LENGTH;
}
```

Comment (noirbizarre) [inline js/components/form/tag-completer.vue]: This is the part that is not working 😞

File: udata/tags.py (hunk 2 above)
Comment (noirbizarre) [inline udata/tags.py]: Updated for a more explanatory comment

File: js/components/form/tag-completer.vue (hunk 1 above)
Comment (jdesboeufs) [inline js/components/form/tag-completer.vue]: Comparison operators cannot be chained in JS.
Each part return a `boolean`. And you don't want to compare a `boolean` with the rest of your expression.

**Pull request merged**